src/js/mocks/server.js

```diff
@@ -0,0 +1,105 @@
+const jsonServer = require('json-server')
+const fs = require('fs');
+const server = jsonServer.create()
+const router = jsonServer.router('db.json')
+const middlewares = jsonServer.defaults()
+
+const VALID_USER = 'user';
+const VALID_PASSWORD = 'pass';
+const ACCESS_TOKEN = 'sdfjknwjnfwjnofonwewwf';
+const REFRESH_TOKEN = 'sdfjknwjnfwjnofonwewwf';
+
+
+server.use(middlewares)
+server.use(jsonServer.bodyParser)
+
+server.post('/api/accounts/login', (req, res) => {
+	const response = {detail: "Login or password invalid."}
+
+	const login = req.body.login;
+	const password = req.body.password;
+
+	if (login !== VALID_USER || password !== VALID_PASSWORD){
+		res.status(400).json(response)
+		return
+	}
+	res.status(200).json(
+		{
+			token: {
+				access: ACCESS_TOKEN,
+				refresh: REFRESH_TOKEN
+			}
+		}
+	)
+})
+
+server.get('/api/wagtail/v2/pages/', (req, res)=>{
+	try{
+		const content = require('./services.json');
+		res.status(200).json(content)
+	} catch{
+		res.status(500).json({error: 'Error reading content.'})
+	}
+})
```

Comment:
could you apply the linter to your code? for example here is tabs instead of spaces, and there should be a space between `try` and `{`